src/ol_infrastructure/substructure/consul/__main__.py

```diff
@@ -13,3 +17,26 @@
         datacenters=["operations-ci", "operations-qa", "operations"]
     ),
 )
+
+operations_log_service_query = PreparedQuery(
+    "operations-log-service-query",
+    name="logging",
+    service="${match(2)}",
+    tags=["logging"],
+    failover=PreparedQueryFailoverArgs(
+        datacenters=["operations-ci", "operations-qa", "operations"]
+    ),
+    template=PreparedQueryTemplateArgs(
+        regexp="^(operations|logging)-(.*?)$", type="name_prefix_match"
+    ),
+)
+
+nearest_service_query = PreparedQuery(
```

Comment:
Not sure what the intended use of this is

Reply:
That is for cutting down on cross-AZ network requests for e.g. Elasticsearch or Mongo from a client